src/components/PublicExplorerListButton.js

```diff
@@ -4,12 +4,14 @@ import { Image, Linking } from 'react-native';
 import icShareActive from '../assets/icons/icShareActive.png';
 import { ListButton } from './HathorList';
 import { COLORS } from '../styles/themes';
+import { useSelector } from 'react-redux';
 
 export function PublicExplorerListButton(props) {
   const { txId } = props;
   const explorerIcon = <Image source={icShareActive} width={24} height={24} />;
+  const explorerUrl = useSelector((state) => state.networkSettings.explorerUrl);
   // XXX: maybe we should have this on the constants or utils to check the network
-  const explorerLink = `https://explorer.hathor.network/transaction/${txId}`;
+  const explorerLink = `${explorerUrl}transaction/${txId}`;
```

Comment:
I think we could use 

```
new URL(`transaction/${txId}`, `${explorerUrl}`).href
```

This way we will prevent any errors if the user sets the explorer URL without the last `/`.

What do you think?

Reply:
I realize that URL is provided by Node, but it is not provided by Hermes, the ReactNative engine. We have two options, either verify the `/` in the end, or install a polyfill lib. I prefer the first option, to verify and adapt.

Reply:
I'm ok with both options

Reply:
This is sad :(

But I prefer we avoid installing new packages just because of that. I saw that axios has a helper to do it but we don't have this in the mobile, only in the lib.

You can create a method exactly the same for our code and use it: https://github.com/axios/axios/blob/fe7d09bb08fa1c0e414956b7fc760c80459b0a43/lib/helpers/combineURLs.js

Reply:
- added combineURLs method: 8b8b716180d0f19043f19ab49debb10a0f91dbcd
- refactored URLs combination: f7fd7aecb0657269e315f71a38a8d449480cde7f